src/wallet/init.cpp

```diff
@@ -221,11 +220,17 @@ bool InitLoadWallet()
             }
         }
 
-        CWallet * const pwallet = CWallet::CreateWalletFromFile(walletFile);
+        bool fFirstRun;
+        CWallet * const pwallet = CWallet::CreateWalletFromFile(walletFile, fFirstRun);
         if (!pwallet) {
             return false;
         }
         vpwallets.emplace_back(pwallet);
+
+        if (fFirstRun) {
+            // Initial backup
+            return AutoBackupWallet(walletFile, strWarning, strError);
```

Comment:
This makes the multiwallet test fail. We are inside a loop `for (const std::string& walletFile : gArgs.GetArgs("-wallet"))`.
If we `return`, then only the first wallet is processed.

Reply:
Also we ignore `strWarning`/`strError` if set.

Reply:
yeah, will add a log here instead of the return and done. No need to overthink this much.